packages/backend-common/src/s3.ts

```diff
@@ -32,15 +31,29 @@ export const getSignedUrl = (
 		getS3Client(region, useAccelerateEndpoint),
 		new PutObjectCommand({
 			Bucket: bucket,
-			Key: id || uuid4(),
+			Key: id,
 			Metadata: {
 				'user-email': userEmail,
-				'file-name': fileName,
 			},
 		}),
 		{ expiresIn }, // override default expiration time of 15 minutes
 	);
 
+export const getSignedDownloadUrl = async (
```

Comment:
slightly out of the scope of this pr but I think worth doing - could generate the s3 client at the top level in the various index.ts files and then pass it through to this function and getSignedUploadurl.

You could potentially reduce code duplication even more by having just one function that either takes a GetObjectCommand or a PutObjectCommand